### DISCUSSION TITLE
I want to update to 9.12, npmjs.com is still on 9.11.1

### DISCUSSION THREAD

**Discussion by vanling** [directus/directus#13696] [Archive]
Any ideas/workarounds or tips to update directus while [npmjs.com](https://www.npmjs.com/package/directus) is not yet updated?

❤️❤️❤️ I really wanna see the new data flows ❤️❤️❤️

**Comment by vanling** [comment]
Not the only one, and I guess we should just wat a little bit longer. 
https://github.com/directus/directus/issues/13693

Release workflow is failing: https://github.com/directus/directus/actions/runs/2431275139

**Comment by vanling** [comment]
@rijkvanzanten i see its about a memory issue, i found this one https://github.com/pierotofy/set-swap-space adding swapspace, since workflows can not get more memory i guess.

**Comment by Pachat** [comment]
No more. It works now. Flows are great ! Thanks to the developers. I'll sponsor :)
